cypress-tests/cypress/e2e/RoutingUtils/Commons.js

```diff
@@ -29,23 +21,41 @@ export const connectorDetails = {
       body: {},
     },
   },
-  card_pm: {
-    Confirm: {
-      Request: {
-        payment_method_data: {
-          card: card_data,
-        },
-        currency: "USD",
-        customer_acceptance: null,
-        setup_future_usage: "on_session",
+  volumeBasedRouting: {
+    Request: {
+      name: "volume routing",
+      description: "some desc",
+      algorithm: {
+        type: "volume_split",
+        data: [],
       },
-      Response: {
-        status: 200,
-        body: {
-          status: "succeeded",
-          connector: "stripe",
-        },
+      profile_id: "{{profile_id}}",
```

Comment:
Wont js throw invalid string error in here for using special character?  
Asking so because I've faced this before in fixtures.

Reply:
No its throwing any error